lib/extensions.js

```diff
@@ -379,6 +403,45 @@ module.exports = function(realmConstructor) {
                     return classPermissions[0];
                 }
             },
+
+            subscriptions(arg) {
+                let all_subscriptions = this.objects('__ResultSets');
+                if (arg) {
+                    if (arg.includes('*') || arg.includes('?')) {
+                        all_subscriptions = all_subscriptions.filtered(`name LIKE '${arg}'`);
+                    } else {
+                        all_subscriptions = all_subscriptions.filtered(`name == '${arg}'`);
+                    }
+                }
+                let listOfSubscriptions = [];
```

Comment:
So you went with a simple wrapper class after all? Wasn't part of the original use case to get notifications for the Subscription state, which seems impossible on the objects returned here?  Not saying this cannot be done, but you just need to make it _very_ clear in docs, that this is just an immutable snapshot that doesn't change.

Reply:
The use case we have - exposing the existing subscriptions via GraphQL - doesn't require notifications. I'm fine with both but considering this is mainly catering to an internal use case, I'd say 🚢 it as it is and let's improve and align with other SDKs later. I'm also okay with leaving the API undocumented or documented with a beta label if we feel we'll want to change them soon.

Reply:
Yeah, I plan to wrap up and ship it as 2.19.0-rc.2.